README.md

```diff
@@ -52,12 +56,14 @@ analysis-runner \
     --dataset <dataset> \
     --description <description> \
     --access-level <level> \
-    --output-dir gs://<bucket-path> \
+    --output-dir <directory-within-the-bucket> \
```

Comment:
Isn't this the directory for all buckets?

Reply:
Good point! Changed to `directory-within-bucket`.